src/main/java/io/stargate/sgv2/jsonapi/service/projection/DocumentProjector.java

```diff
@@ -84,11 +83,8 @@ public static DocumentProjector createFromDefinition(
       return defaultProjector();
     }
     if (!projectionDefinition.isObject()) {
-      throw new JsonApiException(
-          ErrorCode.UNSUPPORTED_PROJECTION_PARAM,
-          ErrorCode.UNSUPPORTED_PROJECTION_PARAM.getMessage()
-              + ": definition must be OBJECT, was "
-              + projectionDefinition.getNodeType());
+      throw ErrorCode.UNSUPPORTED_PROJECTION_PARAM.toApiException(
```

Comment:
Using newer `ErrorCode.toApiException()` as more compact, templatised variant, to make later refactoring simpler.